### PR TITLE
Attempt retries in supplemental image mirroring

### PR DESCRIPTION
**Pull request by smg247**
There have been some failures recently in the `periodic-image-mirroring-supplemental-ci-images` job due to flakes. To cut down on these we can borrow the retry idea from `periodic-image-mirroring-openshift`

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]

**Review by petr-muller — COMMENTED** [review]
/hold

File: ci-operator/jobs/infra-image-mirroring.yaml [hunk 1]
```diff
@@ -959,10 +959,16 @@ periodics:
         failures=0
         for mapping in /etc/imagemirror/mapping_supplemental_ci_images_*; do
           echo "Running: oc image mirror -f=$mapping --skip-multiple-scopes"
-          if ! oc image mirror -f="$mapping" --skip-multiple-scopes; then
-            echo "ERROR: Failed to mirror images from $mapping"
-            failures=$((failures+1))
-          fi
+          attempts=3
+          for attempt in $( seq $attempts ); do
+            if ! oc image mirror -f="$mapping" --skip-multiple-scopes; then
```
Comment (petr-muller) [inline ci-operator/jobs/infra-image-mirroring.yaml:964]: We need to break on success, not on failure

Comment (smg247) [reply ci-operator/jobs/infra-image-mirroring.yaml:964]: ...oops, good catch!

**Review by petr-muller — APPROVED** [review]

**Pull request merged**